docs/exclusive-content.md

```diff
@@ -6,16 +6,16 @@ sidebar_label: Exclusive Content
 
 One of the perks of Web Monetization is that its JavaScript API can be used to
 make your page respond to Web Monetization. You can reward the people who
-support your site by giving Web Monetized viewers exclusive content.
+support your site by giving Web-monetized viewers exclusive content.
```

Comment:
I know it's probably correct, but "Web-monetized" just looks off to me :)

Reply:
After the new Coil pages are up, I'll need to edit the web mon docs. I want it to be consistent with what Fabian shows. I'm pretty sure that includes dropping the hyphen for that exact reason. It's usually correct to include, but doesn't look all that great.